src/credential/credential.controller.ts

```diff
@@ -21,9 +21,10 @@ export class CredentialController {
     return this.credentialService.createCredential(createCredentialDto);
   }
 
-  @Post(':id')
-  updateCredential(@Param('id') id: string, @Body() createCredentialDto: CreateCredentialDto): Promise<Credential> {
-    return this.credentialService.updateCredential(id, createCredentialDto);
+  // Note: CreateCredentialDto is used here but only the revocation specific fields are required and used by this method.
```

Comment:
Probably good to have a different DTO then. Otherwise you might be restrained in the class-validations that you can use on CreateCredentialDto. Don't need to make this change now if you don't want to, of course, since you aren't using any validations yet.